src/schematools/cli.py

```diff
@@ -515,7 +516,7 @@ def introspect_db(
 @introspect.command("geojson")
 @click.argument("dataset_id")
 @click.argument("files", nargs=-1, required=True)
-def introspect_geojson(dataset_id: str, files: Iterable[str]) -> None:
+def introspect_geojson(dataset_id: str, files: List[str]) -> None:
```

Comment:
dit kunnen we straks met py-upgrade nogmaals fixen om te veranderen in `list[str]` (python 3.9 notatie).